debian/precise/foreman/foreman-debug.dirs

```diff
@@ -0,0 +1 @@
+usr/share/foreman/script/foreman-debug.d/
```

Comment:
I'm not 100% sure that an `install` entry without a `dirs` entry will work as described, but it's fairly likely. we can test it :)